File: shanny-blog-java/blog-pojo/src/main/java/com/shanny/dto/AboutDTO.java
package com.shanny.dto;

import com.shanny.enums.AboutEnum;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Schema(description = "添加作者信息提交的数据")
public class AboutDTO {
    private Long id;
    private String avatar;
    private String name;
    private String introduce;
    private String tag;
    private String github;
    private String steam;
    private String web;
    private String biliBili;
    private Boolean isActive;
    private String other;
}

File: shanny-blog-net/blog-pojo/Dtos/AboutDTO.cs
namespace blog_pojo.Dtos
{
    public class AboutDTO
    {
        public long Id { get; set; }
        public string Avatar { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Introduce { get; set; } = string.Empty;
        public string Tag { get; set; } = string.Empty;
        public string Github { get; set; } = string.Empty;
        public string Steam { get; set; } = string.Empty;
        public string Web { get; set; } = string.Empty;
        public string BiliBili { get; set; } = string.Empty;
        public bool? IsActive { get; set; }
        public string Other { get; set; } = string.Empty;
    }
}
